build.sbt

```diff
@@ -323,19 +320,32 @@ lazy val telnetd = project
   .settings(commonSettings)
   .settings(
     libraryDependencies ++= Seq(
-      "org.tpolecat" %% "tuco-core" % tucoVersion,
+      "org.tpolecat" %% "tuco-core"  % tucoVersion,
       "org.tpolecat" %% "tuco-shell" % tucoVersion
     ),
-    dockerExposedPorts  := List(6666),
-    dockerRepository    := Some("geminihlsw")
+    packageName in Docker := "gem",
+    dockerBaseImage       := "openjdk:8u141",
+    dockerExposedPorts    := List(6666),
+    dockerRepository      := Some("sbfocsdev-lv1.cl.gemini.edu"),
+    dockerLabels          := imageManifest.labels,
+
+    // Install nc before changing the user
+    dockerCommands       ++= dockerCommands.value.flatMap {
+      case c @ Cmd("USER", args @ _*) =>
+        Seq(
+          ExecCmd("RUN", "apt-get", "update"),
+          ExecCmd("RUN", "apt-get", "--assume-yes", "install", "netcat-openbsd"),
+          c
+        )
+      case cmd => Seq(cmd)
+    }
```

Comment:
The idea here is to use netcat to probe the telnet port to see if it's open, as a health check. But it leaves the telnet server in a weird state and I haven't been able to figure out why. So for now this is unused and will probably go away. When we add the web service to the app we can just use curl for the health check.